### ISSUE TITLE
[a11y]: FullPage Error, Screen Reader Announces Unnecessary Pointer Description as "Straight Right Arrow"

### ISSUE THREAD

**Issue by anamikaanu96** [carbon-design-system/ibm-products#6233]
### Package

@carbon/ibm-products

### Browser

Firefox

### Operating System

Windows

### Package version

v2.52.0-rc.0

### Automated testing tool and ruleset

Manual testing

### Assistive technology

JAWS

### Description

The screen reader is announcing the pointer indicating an error as "straight right arrow," which is not necessary or relevant for users relying on assistive technologies. This announcement can create confusion and clutter the auditory experience.
The screen reader should not announce visual indicators like pointers unless they provide essential information relevant to the user. 

### Reproduction/example

https://ibm-products.carbondesignsystem.com/?path=/story/ibm-products-components-full-page-error-fullpageerror--full-page-error&globals=viewport:basic

### Steps to reproduce

This was reported in all variants of the Full Page Error.

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/ibm-products/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/ibm-products/issues) for duplicate problems
